hca-api-server/src/server.js

```diff
@@ -0,0 +1,5 @@
+const app = require('./app');
+
+app.listen(3000, () => {
```

Comment:
Might be nice to use https://www.npmjs.com/package/portfinder.  One less thing that can go wrong in the dev environment.